var/spack/repos/builtin/packages/py-radical-entk/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyRadicalEntk(PythonPackage):
+    """RADICAL Ensemble Toolkit is used for developing and executing
+    large-scale ensemble-based workflows."""
+
+    homepage = 'https://radical-cybertools.github.io'
+    git      = 'https://github.com/radical-cybertools/radical.entk.git'
+    pypi     = 'radical.entk/radical.entk-1.6.7.tar.gz'
+
+    maintainers = ['andre-merzky']
+
+    version('devel', branch='devel')
```

Comment:
```suggestion
    version('develop', branch='devel')
```
Spack has a hard-coded list of versions considered to be newer than stable releases (develop, main, master, trunk).